raiden/ui/app.py

```diff
@@ -267,8 +267,8 @@ def run_app(
 
     config['chain_id'] = given_network_id
 
-    log.debug('Network type', type=network_type)
-    if network_type == 'main':
+    log.debug('Environment setting', type=environment)
+    if environment == Environment.PRODUCTION:
         config['network_type'] = NetworkType.MAIN
```

Comment:
I'd suggest to rename the config variable and the constant to `environment` as well, to keep things consistent.

Reply:
I added a comment to clarify this. `environment` is meant to be a more general configuration including the network type but also not directly related settings like if Matrix rooms have to be private.